src/niscope/metadata/functions_addon.py

```diff
@@ -3,147 +3,3 @@
 
 functions_override_metadata = {
 }
-
-functions_additional_get_self_cal_last_date_and_time = {
```

Comment:
That seems incorrect. I moved the information into the internal metadata in the addition file, which is where it's supposed to go -- the addon.py files, according to Mark Silva, are intended to be for quick fixes that weren't able to be pushed into the internal metadata and exported. These dicts were deleted since they now exist internally and keeping them in functions_addon.py is redundant. I might be mistaken, though, let me know if I've gotten that wrong.

Reply:
Ah, sorry I missed that. Ignore my comment.